cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3990,6 +3990,12 @@ TEST(TestArrowReaderAdHoc, CorruptedSchema) {
   TryReadDataFile(path, ::arrow::StatusCode::IOError);
 }
 
+TEST(TestArrowReaderAdHoc, InvalidRepetitionLevels) {
+  // GH-45185 - Repetition levels start with 1 instead of 0
+  auto path = test::get_data_file("ARROW-GH-45185.parquet", /*is_good=*/false);
+  TryReadDataFile(path, ::arrow::StatusCode::IOError);
```

Comment:
Would you mind also check the status is "The repetition level at the start of a record must be 0 but got ..."

Reply:
:+1: done